config/environment.js

```diff
@@ -151,6 +153,9 @@ module.exports = function(environment) {
       id: 467005,
       enabled: environment === 'production',
       forceSSL: true
+    },
+    alienDom: {
+      keepInDom: `${ALIEN_DOM_ROOT} > :not(.ember-view):not(#fb-root), ${ALIEN_DOM_ROOT} > head > link[rel=stylesheet]:not([href*=assets])`
```

Comment:
oy! yes. you're right. I went one negative too far with the this config key. I'll change this.